itest/starboard/starboard_cli_test.go

```diff
@@ -61,6 +64,63 @@ func NewPodSpec(podName string, containers map[string]string, args ...string) (*
 	return pod, err
 }
 
+func IsVulnerabilityReportForContainerOwnedBy(containerName string, owner client.Object) types.GomegaMatcher {
+	return &vulnerabilityReportMatcher{
+		containerName: containerName,
+		owner:         owner,
+	}
+}
+
+type vulnerabilityReportMatcher struct {
+	owner         client.Object
+	containerName string
+
+	failureMessage        string
+	negatedFailureMessage string
+}
+
+func (matcher *vulnerabilityReportMatcher) Match(actual interface{}) (success bool, err error) {
+	_, ok := actual.(v1alpha1.VulnerabilityReport)
+	if !ok {
+		return false, fmt.Errorf("vulnerabilityReportMatcher expects a VulnerabilityReport")
+	}
+	gvk, err := apiutil.GVKForObject(matcher.owner, starboard.NewScheme())
+	if err != nil {
+		return false, err
+	}
+	m := MatchFields(IgnoreExtras, Fields{
```

Comment:
For my understanding: What's MatchFields? Where is it defined?

Reply:
With Ginkgo / Gomega a common approach is to use dot imports - see imports sections:

```go
import (
	. "github.com/onsi/ginkgo"
	. "github.com/onsi/gomega"
	. "github.com/onsi/gomega/gbytes"
	. "github.com/onsi/gomega/gstruct"
)
```

This is syntactic sugar, you could also [avoid dot imports](https://onsi.github.io/ginkgo/#avoiding-dot-imports).

Reply:
Ah lazy me should've opened the PR in an IDE to realize it was a dot import. That makes a lot more sense!